src/main/java/org/syncany/plugins/dropbox/DropboxTransferManager.java

```diff
@@ -389,4 +389,12 @@ public boolean testRepoFileExists() {
 			return false;
 		}
 	}
+
+	private URI makeChildURI(URI parent, String child) {
+		if (!parent.toString().endsWith("/")) {
+			parent = URI.create(parent.toString() + "/");
+		}
+		return parent.resolve(child);
```

Comment:
You know I'm a fan of spaces and newlines. Would you mind adding one before the `return`?